src/containers/SideBar/components/Settings/Markups/CpuMiningMarkup.tsx

```diff
@@ -1,47 +1,32 @@
 import { useCallback } from 'react';
-import { invoke } from '@tauri-apps/api/tauri';
-import { useAppStatusStore } from '@app/store/useAppStatusStore.ts';
 import { useAppStateStore } from '@app/store/appStateStore.ts';
-import { useCPUStatusStore } from '@app/store/useCPUStatusStore.ts';
-import { useGPUStatusStore } from '@app/store/useGPUStatusStore.ts';
 import { useMiningStore } from '@app/store/useMiningStore.ts';
-import { useShallow } from 'zustand/react/shallow';
 import { Typography } from '@app/components/elements/Typography.tsx';
 import { ToggleSwitch } from '@app/components/elements/ToggleSwitch.tsx';
 import { MinerContainer } from '../../../Miner/styles';
 import { useTranslation } from 'react-i18next';
+import { useAppConfigStore } from '@app/store/useAppConfigStore';
 
 const CpuMiningSettings = () => {
     const { t } = useTranslation(['common', 'settings'], { useSuspense: false });
+    const isCPUMining = useMiningStore((s) => s.cpu.mining.is_mining);
+    const isGPUMining = useMiningStore((s) => s.gpu.mining.is_mining);
+    const isCpuMiningEnabled = useAppConfigStore((s) => s.cpu_mining_enabled);
+    const setCpuMiningEnabled = useAppConfigStore((s) => s.setCpuMiningEnabled);
```

Comment:
might want to also set `miningControlsEnabled` when updating gpu/cpu_mining_enabled in `useAppConfigStore` - i turned both off in settings and got stuck in 'start mining' witht he button disabled but obviously mining will never start 😅

Reply:
Disabled mining button when both cpu and gpu mining are disabled in settings